onnxruntime/core/providers/migraphx/migraphx_execution_provider.cc

```diff
@@ -426,8 +413,6 @@ static bool getMIGraphXType(ONNXTensorElementDataType type,
       mgx_type = migraphx_shape_int64_type;
       break;
     case ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT4:
-      mgx_type = migraphx_shape_uint8_type;
```

Comment:
Add a comment here if we're going to flow through so its clear this is "flow through" to the next type

Reply:
I will make a note that it is intentional. The clang-tidy reported that those two branches are "identical".